### ISSUE TITLE
Kernel 3.4 not enable LCD screen

### ISSUE THREAD

**Issue by mortaromarcello** [linux-sunxi/linux-sunxi#110]
The kernel does not enable the LCD screen of my tablet (mediacom 711i)
options CONFIG_LYCHEE_LCD_SUN4I=y CONFIG_LYCHEE_HDMI_SUN4I=y  are ignored by the .config file.

**Comment by amery** [comment]
can you test the sunxi-v3.0.52-r2 and sunxi-v3.0.52-r1 tags?

**Comment by mortaromarcello** [comment]
Yes, it works. But version 3.4 has the modules dvb-usb that I would need.

**Comment by amery** [comment]
then, can you test the sunxi-v3.4.19-r2 and sunxi-v3.4.19-r1 tags? ;-)

**Comment by mortaromarcello** [comment]
Even with sunxi-v3.4.19-r2 unfortunately the LCD screen does not work.:-(

**Comment by amery** [comment]
and -r1?

**Comment by mortaromarcello** [comment]
comparing drivers/video/sunxi/ directory I saw that in version 3.4 is missing Kconfig file.
